### ISSUE TITLE
Furious Focus Bonus Appears but is Not Added

### ISSUE THREAD

**Issue by Envibel** [WittleWolfie/CharacterOptionsPlus#60]
Furious focus is appearing in the attack roll in the combat log but is not actually being added to the total. In the attached screenshot the total bonus should be +11, not +9.

![image](https://user-images.githubusercontent.com/113065492/189454026-c8492341-6989-4d1e-b492-e439d3dbcf71.png)

**Comment by Envibel** [comment]
Here's another screenshot showing that it's not being invisibly added either. Here if the +2 from Furious Focus was actually being added this attack would've hit 
![image](https://user-images.githubusercontent.com/113065492/189454667-92911c88-199c-454f-a22d-0b63035520a0.png)

**Comment by WittleWolfie** [comment]
Nice catch. I swear I had tested that at some point but I actually built this like a year ago.

Anyways, fixed it!

![image](https://user-images.githubusercontent.com/399167/189456533-f7de39cb-17f8-41d3-996f-d2ddf3e0815b.png)

Pushing a release shortly.

If you're curious the issue: to identify that PowerAttack was applied we use `OnEventDidTrigger`, but at that point it already has calculated the result. All we did was add the modifier for the tooltip.

The change is to modify the event result: `evt.Result -= powerAttackModifier.Value.Value;`
